csrc/flashinfer_rope_ops.cu

```diff
@@ -37,7 +37,7 @@ void apply_llama31_rope_pos_ids(at::Tensor q, at::Tensor k, at::Tensor q_rope, a
                                 int64_t cuda_stream);
 
 void apply_rope_pos_ids_cos_sin_cache(at::Tensor q, at::Tensor k, at::Tensor q_rope,
-                                      at::Tensor k_rope, at::Tensor cos_cache, at::Tensor sin_cache,
+                                      at::Tensor k_rope, at::Tensor cos_sin_cache,
```

Comment:
Please also update the correspondingly entry in `csrc/flashinfer_ops.cu` which in the pybind file for AOT mode.

Reply:
done!